neat/channels/compilechannels

```diff
@@ -1,14 +1,27 @@
 #!/usr/bin/env python3
-import os, glob, sys, inspect, shutil, subprocess
+import imp, os, glob, sys, inspect, shutil, subprocess
+
+# get new paths list without the current working directory
```

Comment:
interesting approach, mabye put this into a function, e.g., `import_neat_from_nonlocal_path`?

Reply:
put the import in a function